static/app/views/settings/organizationAuditLog/auditLogList.tsx

```diff
@@ -19,20 +21,23 @@ const avatarStyle = {
 };
 
 type Props = {
-  entries: any[];
-  pageLinks: string;
+  isLoading: boolean;
+  entries: AuditLog[] | null;
+  pageLinks: string | null;
   eventType: string;
   eventTypes: string[];
   onEventSelect: (value: string) => void;
 };
 
 const AuditLogList = ({
+  isLoading,
   pageLinks,
   entries,
   eventType,
   eventTypes,
   onEventSelect,
 }: Props) => {
+  const userOptions = ConfigStore.get('user')?.options ?? {};
```

Comment:
I wonder if we should replace stuff like this with `useConfig`?

Reply:
@evanpurkhiser yes please